code/datums/status_effects/debuffs.dm

```diff
@@ -126,21 +126,22 @@
 /datum/status_effect/incapacitating/sleeping/tick()
 	if(owner.maxHealth)
 		var/health_ratio = owner.health / owner.maxHealth
-		var/healing = -0.2
+		var/healing = -0.003125 //set for a base of 0.25 healed per 2-second interval asleep in a bed with covers.
 		if((locate(/obj/structure/bed) in owner.loc))
-			healing -= 0.3
+			healing -= 0.00625
 		else if((locate(/obj/structure/table) in owner.loc))
-			healing -= 0.1
+			healing -= 0.003125
 		for(var/obj/item/bedsheet/bedsheet in range(owner.loc,0))
 			if(bedsheet.loc != owner.loc) //bedsheets in your backpack/neck don't give you comfort
 				continue
-			healing -= 0.1
+			healing -= 0.003125
 			break //Only count the first bedsheet
-		if(health_ratio > 0.8)
+		if(health_ratio > -0.5)
```

Comment:
All of these changed number are magic numbers and need to be behind defines

Reply:
As... opposed to the previous magic numbers?

Reply:
Why? What's the gain? I can do the math in the var easy enough, but it's used just one place AFAIK.